product/dialogs/miq_dialogs/miq_provision_vmware_dialogs_clone_to_template.yaml

```diff
@@ -398,6 +398,14 @@
           :required: false
           :display: :edit
           :data_type: :integer
+        :storage_profile:
```

Comment:
@jameswnl I agree with your previous comment that this field should be `:placement_storage_profile` for consistency.  (For the cloud side we dropped the placement, but it is still prevalent here in the infra dialogs.)